db/blip_handler.go

```diff
@@ -440,10 +441,10 @@ func (bh *blipHandler) sendChanges(sender *blip.Sender, opts *sendChangesOptions
 	// On forceClose, send notify to trigger immediate exit from change waiter
 	if forceClose {
 		user := ""
-		if bh.collection.User() != nil {
-			user = bh.collection.User().Name()
+		if bh.db.User() != nil {
+			user = bh.db.User().Name()
 		}
-		bh.collection.DatabaseContext.NotifyTerminatedChanges(user)
+		bh.db.DatabaseContext.NotifyTerminatedChanges(user)
```

Comment:
It's not clear to me why this matters. We copy the DB context after the user has been set on it.

The only change I could see is that we might actually have to call 		bh.db.DatabaseContext.NotifyTerminatedChanges(user) on both collection and db? I'm not sure what NotifyTerminatedChanges does though.

Reply:
For now it doesn't make a difference (db == collection) - but when we have more than one collection, users are stored at the database level and can read/write to multiple collections.

Making this change just covers us when we change `db` to be something other than a user-data collection.